_data/people_list.yml

```
@@ -0,0 +1,618 @@
+  github: yuyichao
+  status: active
+  comment: Who ?
+
+- name: Matthias Zach
+  affiliation: University of Kaiserslautern-Landau
+  website: https://math.rptu.de/ags/agag/personen/mitglieder
+  email: zach@rptu.de
+  github: HechtiDerLachs
+  status: active
+
+- name: Mohamed Barakat
+  affiliation: University of Siegen
+  website: https://mohamed-barakat.github.io/
+  email: mohamed.barakat@uni-siegen.de
+  github: mohamed-barakat
+  status: retired
+
+- name: Reimer Behrends
+  affiliation: University of Kaiserslautern-Landau
+  github: rbehrends
+  status: retired
+  comment: Parallelization, GAP garbage collector interface
+
+- name: Taylor Brysiewicz
+  affiliation: University of Western Ontario
+  website: https://sites.google.com/view/taylorbrysiewicz/home
+  email: tbrysiew@uwo.ca
+  github: tbrysiewicz
+  status: retired
+
+- name: Giovanni De Franceschi
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  comment: Group theory - who ?
+  github: GDeFranceschi
+
+- name: Alexander Dinges
+  affiliation: University of Kaiserslautern-Landau
+  website: https://pl.cs.uni-kl.de/homepage/de/staff/AlexanderDinges/
+  email: alexander.dinges@cs.rptu.de
+  github: AlexD97
+  status: retired
+  comment: Worked for Janko on modules
+
+- name: Raul Epure
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  github: raulepure
+  comment: Used to work in TU KL, under whom and for what ? who ?
+
+- name: Rafael Fourquet
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  comment: General infrastructure who ?
+  github: rfourquet
+
+- name: Sebastian Gutsche
+  affiliation: University of Kaiserslautern-Landau
+  website: https://sebasguts.github.io/
+  status: retired
+  github: sebasguts
+
+- name: William Hart
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  github: wbhart
+  comment: Initial contributor to Oscar.
+
+- name: Florian Heiderich
+  affiliation: University of Siegen
+  website: https://www.heiderich.org/math/
+  status: retired
+  github: heiderich
+
+- name: Fredrik Johansson
+  affiliation: Institut de Mathématiques de Bordeaux
+  website: http://fredrikj.net/
+  email: fredrik.johansson@gmail.com
+  github: fredrik-johansson
+  status: retired
+  comment: The FLINT guy.
+
+- name: Marek Kaluba
+  affiliation: KIT Karlsruhe
+  website: https://kalmarek.github.io/
+  email: kalamar@mailbox.org
+  github: kalmarek
+  status: retired
+  comment: Freelance mathematician, formerly at KIT
+
+- name: Avinash (Avi) Kulkarni
+  affiliation: Dartmouth College
+  website: https://math.dartmouth.edu/~akulkarn/index.html
+  email: avinash.a.kulkarni@dartmouth.edu
+  github: a-kulkarn
+  status: retired
+  comment: Who ?
+
+- name: Frank Lübeck
+  affiliation: RWTH Aachen University
+  website: http://www.math.rwth-aachen.de/~Frank.Luebeck/index.html
+  email: Frank.Luebeck@math.rwth-aachen.de
+  github: frankluebeck
+  status: retired
+
```

I am not aware of any direct contribution by Frank to OSCAR, so perhaps he should be removed here.
```suggestion
- name: Frank Lübeck
  affiliation: RWTH Aachen University
  website: http://www.math.rwth-aachen.de/~Frank.Luebeck/index.html
  email: Frank.Luebeck@math.rwth-aachen.de
  github: frankluebeck
  status: retired
```